packages/terra-table/src/utils/constants.js

```diff
@@ -0,0 +1,7 @@
+const ERRORS = {};
```

Comment:
Do we really need a separate constant file for these errors?

Reply:
This file is not only for errors but for any constants that we may use. Having a separate file would mean a single place to update them instead of having to hunt for them through multiple source files.